### ISSUE TITLE
How to train oneself

### ISSUE THREAD

**Issue by 2640888968** [nagejacob/TBSN#8]
Hello author, may I ask why I removed the pre trained model and directly trained it on the SIDD validation set, which only had 37.51, while using your pre trained test set only had 37.70. I am a first-year graduate student and just started working on this, so I don't understand how to test this SIDD benchmark, which is also the reason why I couldn't reproduce it. Can you answer how I can solve this problem

**Comment by nagejacob** [comment]
You may train on the SIDD training split. The self-supervised training may have variance at different epochs, so validate the checkpoints during training and pick up the best one may help. As for SIDD benchmark, you may refer to [https://abdokamel.github.io/sidd/](url)

**Comment by 2640888968** [comment]
Thank you for taking the time to answer. I am starting from scratch by removing pre training from [https://github.com/nagejacob/TBSN/blob/main/option/tbsn_sidd.json](url) , and I am not sure if this is correct
